beamerthemeuulm.sty

```diff
@@ -198,131 +198,259 @@
+    \thirdpage{#3}%
 }
 
-\newcommand{\leftthenright}[2]{
-    \halfpage{#1}
+\newcommand\leftthenright[2]{%
+    \halfpage{#1}%
     \hfill
-    \onslide<2->{\halfpage{#2}}
+    \onslide<2->{\halfpage{#2}}%
 }
 
-\newcommand{\rightthenleft}[2]{
+\newcommand\rightthenleft[2]{
     \onslide<2->{\halfpage{#1}}
     \hfill
     \halfpage{#2}
 }
 
-\newcommand{\leftmiddlethenright}[3]{
-	\thirdpage{#1}
+\newcommand\leftmiddlethenright[3]{%
+    \thirdpage{#1}%
     \hfill
-    \onslide<2->{\thirdpage{#2}}
+    \onslide<2->{\thirdpage{#2}}%
     \hfill
-    \onslide<3->{\thirdpage{#3}}
+    \onslide<3->{\thirdpage{#2}}%
 }
 
-\newcommand{\rightmiddlethenleft}[3]{
-	\onslide<3->{\thirdpage{#1}}
-    \hfill
-    \onslide<2->{\thirdpage{#2}}
+\newcommand\rightmiddlethenleft[3]{%
+    \onslide<3->{\thirdpage{#1}}%
     \hfill
-    \thirdpage{#3}
+    \onslide<2->{\thirdpage{#2}}\hfill\thirdpage{#2}%
 }
 
-\newcommand{\leftorright}[2]{ % only works in recording mode
-    \ifrecording
-    \onslide<1>{\halfpage{#1}}
-    \onslide<2>{\hfill}
-    \onslide<3>{\halfpage{#2}}
-    \else
-	\leftthenright{#1}{#2}
-	\fi
+\newcommand\leftorright[2]{%
+\ifrecording
+    \onslide<1>{\halfpage{#1}}%
+    \onslide<2>{\hfill}%
+    \onslide<3>{\halfpage{#2}}%
+\else
+    \leftthenright{#1}{#2}%
+\fi
 }
 
-\newcommand{\rightorleft}[2]{ % only works in recording mode
-    \ifrecording
-    \onslide<3>{\halfpage{#1}}
-    \onslide<2>{\hfill}
-    \onslide<1>{\halfpage{#2}}
-    \else
-	\rightthenleft{#1}{#2}
-	\fi
+\newcommand\rightorleft[2]{%
+\ifrecording
+    \onslide<3>{\halfpage{#1}}%
+    \onslide<2>{\hfill}%
+    \onslide<1>{\halfpage{#2}}%
+\else
+    \rightthenleft{#1}{#2}%
+\fi
 }
 
-\newcommand{\leftmiddleorright}[3]{ % only works in recording mode
-    \ifrecording
-    \onslide<1>{\thirdpage{#1}}
-    \onslide<2>{\hfill}
-    \onslide<3>{\thirdpage{#2}}
-    \onslide<4>{\hfill}
-    \onslide<5>{\thirdpage{#3}}
-    \else
-	\leftmiddlethenright{#1}{#2}{#3}
-	\fi
+\newcommand\leftmiddleorright[3]{%
+\ifrecording
+    \onslide<1>{\thirdpage{#1}}%
+    \onslide<2>{\hfill}%
+    \onslide<3>{\thirdpage{#2}}%
+    \onslide<4>{\hfill}%
+    \onslide<5>{\thirdpage{#3}}%
+\else
+    \leftmiddlethenright{#1}{#2}{#3}%
+\fi
 }
 
-\newcommand{\rightmiddleorleft}[3]{ % only works in recording mode
-    \ifrecording
-    \onslide<5>{\thirdpage{#1}}
-    \onslide<4>{\hfill}
-    \onslide<3>{\thirdpage{#2}}
-    \onslide<2>{\hfill}
-    \onslide<1>{\thirdpage{#3}}
-    \else
-	\rightmiddlethenleft{#1}{#2}{#3}
-	\fi
+\newcommand\rightmiddleorleft[3]{%
+\ifrecording
+    \onslide<5>{\thirdpage{#1}}%
+    \onslide<4>{\hfill}%
+    \onslide<3>{\thirdpage{#2}}%
+    \onslide<2>{\hfill}%
+    \onslide<1>{\thirdpage{#3}}%
+\else
+    \rightmiddlethenleft{#1}{#2}{#3}%
+\fi
 }
-% ---------------------------------------------
-
 
-% ---------------------------------------------
-% color boxes
-% ---------------------------------------------
-\newcommand{\mydefinition}[2]{
```

Comment:
Did you really remove this mydefinition, mydefinitiontight, ...? We still need those.

Reply:
Same